apps-rendering/src/components/Headline/GalleryHeadline.tsx

```diff
@@ -28,6 +29,11 @@ const styles = css`
 		${headline.xlarge({ fontWeight: 'bold' })}
 		${grid.span('centre-column-start', 8)}
 	}
+
+	${darkModeCss`
+		background-color: ${neutral[10]};
+		color: ${neutral[86]};
```

Comment:
A couple for the palette? And perhaps the `color` above?

```suggestion
		background-color: ${background.headlineDark(format)};
		color: ${text.headlineDark(format)};
```